config.js

```diff
@@ -12,8 +14,11 @@ module.exports = {
 	{ id: 6, name: 'Pizza Pazza', url: 'http://pizzapazza.sk/?page_id=645', module: 'pizzapazza', post: 'prevadzka2=1' },
 	{ id: 7, name: 'Kamenica - Corleone Pizza', url: 'http://www.pizzacorleone.sk/obedove-menu.html', module: 'kamenica' },
 	{ id: 8, name: 'Numero Uno', url: 'http://www.numerouno.sk/denne-menu/', module: 'numerouno' },
-	{ id: 9, name: 'Engerau restaurant', url: 'https://www.zomato.com/sk/bratislava/engerau-restaurant-petr%C5%BEalka-bratislava-v/denn%C3%A9-menu', module: 'engerau' },
-	{ id: 10, name: 'Lokálka', url: 'http://www.lokalka.sk/kopcianska-2/', module: 'lokalka' }
+	{ id: 10, name: 'Lokálka', url: 'http://www.lokalka.sk/kopcianska-2/', module: 'lokalka' },
+	// jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+	{ id: 9, name: 'Engerau restaurant', url: 'https://developers.zomato.com/api/v2.1/dailymenu?res_id=16507706', webUrl: 'https://www.zomato.com/bratislava/engerau-restaurant-petr%C5%BEalka-bratislava-v/daily-menu', module: 'engerau', headers: { 'user_key' : userKey }, noCache: true },
```

Comment:
this is just performance optimization caching to prevent spaming our 'sources' with requests every time somebody accesses the site, its not persisted anywhere, so we should be fine. I recommend removing this setting and related logic to keep it simple